### ISSUE TITLE
feature request: delete bounding boxes

### ISSUE THREAD

**Issue by NicolaiHarich** [lasmue/annotator#2]
In my opinion it's important to have a delete functionality for annotations. 
Either we want to delete an annotation at a specific frame or all annotations belonging to one track. 

Furthermore there might be situations where a user wants to delete all annotations belonging to a certain range. Maybe the track after a certain frame number is wrong (because the tracking algorithm failed at some point).

So in summary:
- [ ] delete annotation at a certain frame 
- [ ] delete annotations within a certain frame range
- [ ] delete all annotations belonging to an annotated object instance
